resources/views/components/component-heading.blade.php

```diff
@@ -1,6 +1,6 @@
 <div class="w-full py-8 bg-theme-secondary-100">
   <div class="container mx-auto ">
-    <h1 class="mx-4 text-2xl font-bold text-center md:text-4xl md:mx-8 xl:mx-16 text-theme-secondary-900">{{ $title }}</h1>
-    <p class="mx-4 mt-4 font-semibold text-center text-theme-secondary-700 md:mx-8 xl:mx-16">{{ $description }}</p>
+    <h1 class="mx-4 text-3xl font-bold text-center md:text-4xl md:mx-8 xl:mx-16 text-theme-secondary-900">{{ $title }}</h1>
```

Comment:
I think this is now in line with the regular `h1` styling in terms of fonts, can you recheck and remove the explicit fonts being set if that's the cause @leMaur ?